generate/generate.go

```diff
@@ -782,6 +782,25 @@ func (s *service) generateHelperFuncs(a *API) {
 			pn("		return nil, -1, err")
 			pn("	}")
 			pn("")
+			pn("    if l.Count == 0 {")
+			pn("		// look	inside projects ")
+			pn("		p.p[\"projectid\"] = \"-1\"")
+			pn("		l2, err := s.ListVirtualMachines(p)")
+			pn("		if err != nil {")
+			pn("			if strings.Contains(err.Error(), fmt.Sprintf(")
+			pn("				\"Invalid parameter id value=%%s due to incorrect long value format, \"+")
+			pn("				\"or entity does not exist\", id)) {")
+			pn("				return nil, 0, fmt.Errorf(\"No match found for %%s in any project: %%+v\", id, l)")
+			pn("			}")
+			pn("			return nil, -1, err")
+			pn("		}")
+			pn("		if l2.Count == 0 {")
```

Comment:
you can skip the rest of this function as you reused the existing variables, so you can just continue here to the original code...